hw3/01-characters.js

```diff
@@ -1,4 +1,63 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters/rgdr";
```

Comment:
Change this back to the correct url or else an error occurs

```suggestion
const url = "https://thronesapi.com/api/v2/Characters";
```